tools/build_coreclr.bat

```diff
@@ -6,7 +6,7 @@ call %REPO_ROOT%\tools\setenv.bat
 if %errorlevel% neq 0 exit /b %errorlevel%
 
 setlocal
-set TRINITY_CORE_SLN=%REPO_ROOT%\src\Trinity.Core\Trinity.Core.NETStandard.sln
```

Comment:
Could you please elaborate why favor netcore over netstandard?

Reply:
Because Serialize.Linq does not provide a netstandard compatible version, so if we add it as dependency for Trinity.Core, then it will fallback to netframework 4.6 version, that will cause a warning. 

And i know that it is not a good idea to add Serialize.Linq as a dependency for Trinity.Core, as it did not need it at all, but we have to or Serialize.Linq will throw an exception that Action cannot be serialized.